src/main/webapp/js/override.js

```diff
@@ -1,23 +1,31 @@
 define(['angular'], function(angular) {
-  /* Keep in sync with docs/mardown/configuration.md */
   return angular.module('override', [])
-    // see http://uw-madison-doit.github.io/uw-frame/latest/#/md/configuration for howto
```

Comment:
I personally would find it helpful to keep a link to the documentation here. <https://uw-madison-doit.github.io/uw-frame/configuration.html>